app/javascript/readingList/readingList.jsx

```diff
@@ -214,7 +214,7 @@ export class ReadingList extends Component {
             onClick={this.toggleTag}
           />
 
-          <main className="crayons-layout__content">
+          <main id="articles-list" className="crayons-layout__content">
```

Comment:
I will be changing the `id` attribute for all `<main />` elements in the site to something more generic as the only existing skip link uses an `id` attribute that is specific to the main feed on the home page of a Forem instance.